stubs/flake8-docstrings/flake8_docstrings.pyi

```diff
@@ -0,0 +1,13 @@
+import argparse
+import ast
+from typing import Any, Generator, Type
+
+class pep257Checker:
```

Comment:
`AllError` and `EnvironError` are missing, but those seem like an implementation detail. It's probably easiest to mark the file as incomplete (see CONTRIBUTING.md for what that means).